src/containers/ArticlePage/FrontpageArticlePage/components/FrontpageArticlePanels.tsx

```diff
@@ -34,10 +34,10 @@ const FrontpageArticlePanels = ({ article, articleHistory, articleLanguage }: Pr
   const copyrightFields = useMemo<(keyof IArticle)[]>(() => ["copyright"], []);
 
   return (
-    <FormAccordions
+    <FormAccordionsWithComments
       defaultOpen={["frontpage-article-content"]}
       articleId={article?.id}
-      articleType={article?.articleType}
+      articleType="frontpage-article"
```

Comment:
Er det en grunn til at vi har hatt `article.articleType` her før?

Reply:
Usikker, jeg tror ikke det har noe å si hva vi går for, men skal ikke være påståelig 😱